src/main/java/spoon/reflect/code/CtLambda.java

```diff
@@ -60,6 +62,12 @@
 	CtExpression<T> getExpression();
 
 	/**
+	 * @return a method which is implemented by this lambda expression
```

Comment:
@return the method that this lambda expression implements. Must be defined as a non-default method in an interface, eg Consumer.accept().